restapi/src/main/java/io/stargate/web/docsapi/service/ReactiveDocumentService.java

```diff
@@ -442,31 +536,13 @@ public ReactiveDocumentService(
                   documentId,
                   subPathProcessed,
                   rows,
+                  ttl,
                   db.treatBooleansAsNumeric(),
                   context);
             })
         .map(any -> new DocumentResponseWrapper<>(documentId, null, null, context.toProfile()));
   }
 
-  /**
-   * Deletes a document with given ID in the given namespace and collection.
-   *
-   * @param db {@link DocumentDB} to write in
-   * @param namespace Namespace
-   * @param collection Collection name
-   * @param documentId The ID of the document to delete
-   * @param context Execution content @@return Flag representing if the operation was success.
-   */
-  public Single<Boolean> deleteDocument(
-      DocumentDB db,
-      String namespace,
-      String collection,
-      String documentId,
-      ExecutionContext context) {
-    List<String> subPath = Collections.emptyList();
-    return deleteDocument(db, namespace, collection, documentId, subPath, context);
-  }
-
```

Comment:
wait why is this deleted now?

Reply:
The method is never used...all calls currently are using line 467's `deleteDocument` directly